public/plugin.tsx

```diff
@@ -79,6 +80,9 @@ import {
   ObservabilityStart,
   SetupDependencies,
 } from './types';
+import { AccelerationDetailsFlyout } from './components/datasources/components/manage/accelerations/acceleration_details_flyout';
+import { createGetterSetter } from '../../../src/plugins/opensearch_dashboards_utils/public';
+import { toMountPoint } from '../../../src/plugins/opensearch_dashboards_react/public/';
```

Comment:
Do we need to mention add `opensearchDashboardsReact` as a plugin dependency in `dashboards.json` file?

Reply:
`opensearchDashboardsReact` is already a required plugin in `opensearch_dashboards.json`.